Resources/config/config.xml

```diff
@@ -12,6 +12,8 @@
         <parameter key="stampie.mailer.postmark.class">Stampie\Mailer\Postmark</parameter>
         <parameter key="stampie.mailer.mailgun.class">Stampie\Mailer\MailGun</parameter>
         <parameter key="stampie.mailer.mandrill.class">Stampie\Mailer\Mandrill</parameter>
+        <parameter key="stampie.mailer.mailjet.class">Stampie\Mailer\Mailjet</parameter>
+        <parameter key="stampie.mailer.spark_post.class">Stampie\Mailer\SparkPost</parameter>
```

Comment:
Don't create new parameters for this class, as Symfony recommends against this practice. Put the class directly in the abstract services please.

Reply:
Correct. Please submit a new pr where all the existing parameters are removed.